vllm/model_executor/model_loader/loader.py

```diff
@@ -704,13 +711,54 @@ def _get_quantized_weights_iterator(
             model_name_or_path, revision)
 
         quant_state_dict = {}
-        if use_safetensors:
-            weight_iterator = safetensors_weights_iterator(hf_weights_files)
-        else:
-            weight_iterator = pt_weights_iterator(hf_weights_files)
 
-        def generator():
+        def prequant_generator() -> Generator:
```

Comment:
I'm not fond of the term "prequant" here, could it be something along the lines of "quantized_checkpoint"?

Reply:
sure